src/dc_receive_imf.rs

```diff
@@ -1563,13 +1501,13 @@ async fn create_or_lookup_adhoc_group(
         ProtectionStatus::Unprotected,
     )
     .await;
-    for &member_id in &member_ids {
+    for &member_id in member_ids {
```

Comment:
Not that this is wrong, but I'm much more used to the `for member_id in member_ids.iter()` idiom, which is always an iterator over &T and it doesn't consume member_ids.  You don't actually care that this isn't consumed here so it doesn't matter, but I do think it would be more idiomatic.  Up to you though.